Library/Homebrew/cmd/audit.rb

```diff
@@ -676,6 +676,10 @@ def audit_text
     if text =~ /def plist/ && text !~ /plist_options/
       problem "Please set plist_options when using a formula-defined plist."
     end
+
+    if text =~ %r{require "language/go"} && text !~ /go_resource/
```

Comment:
I know. That's why I included parts like “even if that makes for slightly less uniform code”. But really, feel free to ignore this. I just felt the urge to waste some time and express my feelings (again). :wink: